apps/angular/src/app/app.component.html

```diff
@@ -1,2 +1,4 @@
-<camp-app-header></camp-app-header>
-<router-outlet></router-outlet>
+<div class="main-container">
+	<camp-app-header></camp-app-header>
+	<router-outlet></router-outlet>
```

Comment:
You can use self-closing tags.
```suggestion
	<camp-app-header />
	<router-outlet />
```